tests/integration/test_extra.py

```diff
@@ -329,7 +378,7 @@ def test_resolve_toggles(raw_config, expected):
     "raw_config",
     [
         """
-{% if defines.get("toggle_a") or defines.get("toggle_b") %}
+{% if not ( defines.get("toggle_a") and defines.get("toggle_b") ) %}
```

Comment:
we are saying that current implementation supports `not`, `and` and `or` - but it does not

```
def _extract_toggles_from_if_node(if_test_node: nodes.Call) -> Set[str]:
    """Current implementation supports operators `not` and `and`.
    """Current implementation supports operators `not`, `and` and `or`.
    Only following conditions are allowed:
```

Reply:
I guess I need to debug it. In this case, the method throws exception when parentheses are used.

Reply:
I think exception is thrown because we aren't expecting logical ops as a subtree of `not` 
see implementation:
```
    elif isinstance(if_test_node, nodes.Not):
        node = if_test_node.node
```
instead we should recursively extract from deeper nodes
```
    elif isinstance(if_test_node, nodes.Not):
        node = _extract_toggles_from_if_node(if_test_node.node)
``` 

haven't tested it though :) 

also it's not related to your current change, rather smth that should be fixed to make logic with logical ops flexible enough

Reply:
Thanks for the hint! I guess I've fixed it.